packages/idempotency/README.md

```diff
@@ -0,0 +1,83 @@
+# AWS Lambda Powertools for TypeScript
+
+A suite of utilities for AWS Lambda functions to ease the adoption of best practices such as tracing, structured logging, custom metrics, and more.
+
+You can use the library in both TypeScript and JavaScript code bases.
+
+AWS Lambda Powertools for [Python](https://github.com/awslabs/aws-lambda-powertools-python) and  AWS Lambda Powertools for [Java](https://github.com/awslabs/aws-lambda-powertools-java) are also available.
+
+**[📜 Documentation](https://awslabs.github.io/aws-lambda-powertools-typescript/)** | **[NPM](https://www.npmjs.com/org/aws-lambda-powertools)** | **[Roadmap](https://github.com/awslabs/aws-lambda-powertools-roadmap/projects/1)** | **[Examples](https://github.com/awslabs/aws-lambda-powertools-typescript/tree/main/examples)** | **[Serverless TypeScript Demo](https://github.com/aws-samples/serverless-typescript-demo)**
+
+## Table of contents
+
+- [Features](#features)
+- [Getting started](#getting-started)
+  - [Installation](#installation)
+  - [Examples](#examples)
+  - [Serverless TypeScript Demo](#serverless-typescript-demo-application)
+- [Contribute](#contribute)
+- [Roadmap](#roadmap)
+- [Connect](#connect)
+- [Credits](#credits)
+- [License](#license)
+
+## Features

```

Comment:
It could be worth updating all the readme features to include idempotency like in the python readme
https://github.com/awslabs/aws-lambda-powertools-python

Reply:
I was going to hold off on that until we had the idempotency package more fleshed out